conda/linux/AppDir/freecad.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
```

Comment:
`1.0`  ??

Reply:
What is your question? Which version the `.desktop` file should be?

Reply:
This relates to .desktop version, ok..got it!